resources/authgear/templates/en/web/authflowv2/__settings_item.html

```diff
@@ -0,0 +1,52 @@
+<!-- Example usage:
+{{ template "authflowv2/__settings_item.html"
+  (dict
+    "Label" "email"
+    "RedirectURL" "url"
+    "Icon" `email`
+    "IdentityArr" $.Emails
+    "Identity" $.Email
+  )
+}}
+-->
+{{ define "authflowv2/__settings_item.html" }}
+<a
+  class="settings-item with-content row-sep
+    {{ if $.Identity }}
+      with-content
+    {{ else }}
+      without-content
+    {{ end }}
+    "
+  href="{{ $.RedirectURL}}"
+>
+  <i class="alternative-icon material-icons">
+    {{ $.Icon }}
+  </i>
+  <p class="settings-item__label">
+    {{ include (printf "v2.page.settings.default.button-label-%s" $.Label) nil }}
```

Comment:
Render `Label` directly. Caller should be responsible to generate the label content